linux/libfwup.c

```diff
@@ -2272,3 +2272,13 @@ fwup_get_debug_log(char **utf8, size_t *size)
 	*size = vsize >> 1;
 	return 0;
 }
+
+/*
+	fwup_version
+	returns the installed runtime version of libfwupdate
```

Comment:
To match some of the other functions, I think you should add a "Since: " to this comment.